docs/volto_customization/installation.md

```diff
@@ -17,150 +17,124 @@ Getting started with Volto involves setting up a development environment, unders
 
 ## Prerequisites
 
-Before you start working with Volto, ensure you have the following prerequisites:
+Before you start working with this training, ensure you have the following prerequisites:
 
-- <a target="_blank" href="https://nodejs.org/en">Node.js LTS (16.x)</a> - (<a target="_blank" href="https://6.docs.plone.org/install/install-from-packages.html#nvm">see instructions for installation</a>)
+- <a target="_blank" href="https://nodejs.org/en">Node.js LTS (>=20.x)</a> - (<a target="_blank" href="https://6.docs.plone.org/install/install-from-packages.html#nvm">see instructions for installation</a>)
 - <a target="_blank" href="https://www.python.org/">Python</a> - See below for specific versions.
+- <a target="_blank" href="https://pipx.pypa.io/stable/">pipx</a>
+- <a target="_blank" href="https://github.com/nvm-sh/nvm?tab=readme-ov-file#installing-and-updating">nvm</a>
+- <a target="_blank" href="https://www.gnu.org/software/make/">GNU make</a>
 - <a target="_blank" href="https://www.docker.com/get-started">Docker</a> (if using the Plone docker images - <a target="_blank" href="https://6.docs.plone.org/install/containers/index.html">see instructions for installation and usage</a>)
 
 The versions of Python that are supported in Volto depend on the version of Plone that you use.
 
-| Plone | Python       | Volto        |
-| ----- | ------------ | ------------ |
-| 6.0   | 3.8-3.11     | 16.0 or 17.0 |
-| 5.2   | 2.7, 3.6-3.8 | 15.0         |
+| Plone | Python       | Volto          |
+| ----- | ------------ | -------------- |
+| 6.1   | 3.10-3.12    | 18.0           |
+| 6.0   | 3.8-3.12     | 16.0,17.0,18.0 |
+| 5.2   | 2.7, 3.6-3.8 | 15.0           |
 
 Depending on the operating system that you are using, some of the following pre-requisites might change.
 They assume you have a macOS/Linux machine.
 
-## Bootstrap a new Volto project
+## Bootstrap a new Plone stack
 
-To bootstrap a new Volto project, you can use Yeoman [@plone/generator-volto](https://github.com/plone/generator-volto).
-First, install it as a global tool (<a target="_blank" href="https://6.docs.plone.org/volto/recipes/creating-project.html">see instructions for installation</a>):
+To bootstrap a new Plone project(with both backend and frontend), you can use [Cookieplone](https://github.com/plone/cookieplone).
+You can use pipx to run cookieplone (<a target="_blank" href="https://6.docs.plone.org/install/create-project-cookieplone.html#generate-the-project">see instructions for installation</a>):
 
 ```shell
-npm install -g yo
-npm install -g @plone/generator-volto
+pipx run cookieplone project
 ```
 
-Then you can bootstrap the project with:
+You will be presented with a series of prompts. You can also specify the addons you want to install along with the project. You can type the name of the project as `tutorial-project` and name of the addon as `volto-teaser-tutorial` when prompted for the addons.
```

Comment:
Hyphenate "add-on" wherever it occurs in narrative text. The Cookieplone prompt does not hyphenate it. I'll fix that in the template.

```suggestion
You will be presented with a series of prompts.
You can also specify the add-ons you want to install along with the project.
When prompted for {guilabel}`Volto Addon Name`, enter `volto-teaser-tutorial`.
```

This was confusing because you say to change two things here, then later to accept the defaults except for one. I edited the inconsistent sentence.